### PR TITLE
As/bugfix

### PR DESCRIPTION
**Pull request by Adrian-St**

### CODE REVIEW

File: app/assets/javascripts/exercises.coffee [hunk 1]
```diff
@@ -2,58 +2,36 @@
...
+      $("<p id='error' style='color: red'>Title can't be blank</p>").insertAfter(title)
+    document.body.scrollTop = document.documentElement.scrollTop = 0;
+    e.preventDefault()
+    false
 
-SetRatingStar()
 
 loadSelect2 = ->
-
+  console.log("Load call")
```
Comment (tstaubitz) [inline app/assets/javascripts/exercises.coffee]: Please remove the console.log(...)

File: app/assets/javascripts/exercises.coffee [hunk 2]
```diff
@@ -75,6 +53,7 @@ loadSelect2 = ->
       'You can only add 5 topics'
 
 ready =->
+  console.log("Ready Call")
```
Comment (tstaubitz) [inline app/assets/javascripts/exercises.coffee]: same here.

File: app/controllers/exercises_controller.rb [hunk 3]
```diff
@@ -58,26 +58,31 @@ def duplicate
...
 
   # POST /exercises
   # POST /exercises.json
   def create
     @exercise = Exercise.new(exercise_params)
-    @exercise.add_attributes(params[:exercise])
     @exercise.user = current_user
-    exercise_dependencies
+    clone = false
+    clone = true if params[:exercise][:exercise_relation]
```
Comment (tstaubitz) [inline app/controllers/exercises_controller.rb]: this is a little strange to read. I'd rather go for 

if ....
clone = true
else 
clone = false
end

File: app/controllers/exercises_controller.rb [hunk 4]
```diff
@@ -153,24 +158,15 @@ def contribute
...
-  end
+  #def exercise_dependencies
+    #if params[:exercise][:exercise_relation]
+     # @exercise_relation = ExerciseRelation.new
+      #@exercise_relation.clone = @exercise
+     # @exercise_relation.origin_id = params[:exercise][:exercise_relation][:origin_id]
+      #@exercise_relation.relation_id = params[:exercise][:exercise_relation][:relation_id]
+      #@exercise_relation.save
+    #end
+  #end
```
Comment (tstaubitz) [inline app/controllers/exercises_controller.rb]: Please remove commented code

File: app/views/exercises/show.html.slim [hunk 5]
```diff
@@ -156,7 +156,7 @@ table.vertical-table
     tr
       th = t('exercises.show.add_cart')
       td
-        = link_to 'Add to cart', add_to_cart_path(@exercise), class: 'btn btn-main'
+        = link_to 'Add to cart', add_to_cart_path(@exercise), method: "post", class: 'btn btn-main'
```
Comment (tstaubitz) [inline app/views/exercises/show.html.slim]: Please replace hard-coded strings with locales

File: db/seeds.rb [hunk 6]
```diff
@@ -145,10 +148,6 @@
 
 exercise1.update(execution_environment: ee1)
 
-#Exercises from codeocean
-
-java1 = Exercise.create(title: PLATZHALTER , max_rating: 10, private: false, user_id: user1.id )
-
 #exercise2.update(execution_environment: ee1)
 #exercise3.update(execution_environment: ee1)
 #exercise4.update(execution_environment: ee2)
```
Comment (tstaubitz) [inline db/seeds.rb]: Please, remove commented code

File: app/assets/javascripts/exercises.coffee [hunk 7]
```diff
@@ -2,58 +2,35 @@
 # All this logic will automatically be available in application.js.
 # You can use CoffeeScript in this file: http://coffeescript.org/
 
-$star_rating = $('.star-rating .fa')
-
 SetRatingStar = ->
-  $star_rating.each ->
+  $(".star-rating").each ->
     if parseInt($star_rating.siblings('input.rating-value').val()) >= parseInt($(this).data('rating'))
       console.log 'rating1'
```
Comment (rteusner) [inline app/assets/javascripts/exercises.coffee]: remove for production

File: app/assets/javascripts/exercises.coffee [hunk 8]
```diff
@@ -2,58 +2,35 @@
...
       $(this).removeClass('fa-star-o').addClass('fa-star')
     else
       console.log 'rating2'
```
Comment (rteusner) [inline app/assets/javascripts/exercises.coffee]: remove for production

File: db/seeds.rb [hunk 9]
```diff
@@ -58,97 +61,8 @@
 
 AccountLink.create(push_url: 'google.com/pushpush', account_name: 'account1000')
 
-
```
Comment (rteusner) [inline db/seeds.rb]: what was that all about? is it broken? why should it be deleted?

**Pull request merged**